Microsoft.Toolkit.Uwp.Input.GazeControls/GazeScrollbar.xaml.cs

```diff
@@ -0,0 +1,170 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.Toolkit.Uwp.Input.GazeControls
+{
+    /// <summary>
+    /// Gaze optimized scrollbar
+    /// </summary>
+    public sealed partial class GazeScrollbar : UserControl
```

Comment:
Name should be `GazeScrollBar`. Not sure how I feel about re-writing platform controls to have different versions for gaze vs. trying to write helpers for the inbox `ScrollBar` class with potentially alternate styles. As all that's specific to Gaze here is larger hit targets and enabling the existing Gaze helper we have right?

Reply:
I did explore restyling the builtin scrollbar. I even got it to work up to the point of larger hit targets and an always on scrollbar. But there are two problems. 

The first big problem being that even when the gaze is directly on the scrollbar buttons, the VisualTreeHelper functions don't identify that the underlying control as a scroll bar. Instead it returns the underlying GridView. It appears that since the scroll bar is part of the GridView/ScrollView FindElementInHostCoordinates does not return the interaction with the scroll buttons and therefore we cannot control the scrolling with gaze.

The second minor problem is that the system scrollbar includes the scroll thumb which only supports drag operations and takes up valuable interaction space for affecting large changes in the scroll view. 

That is why I chose to implement a gaze specific scrollbar which replaces the scroll thumb with buttons specifically for line-up/line-down and page-up/page-down.

Reply:
>This was the main thing I was trying to say how each of your keyboard layouts become an alternate style (like how we have for the InAppNotification control/sample). We want to avoid reading in loose XAML from a StorageFile for a control library.

> Still looks like everything is based on UserControl and still loading loose XAML. We need to switch things over to Templated Controls and applying styles for the keyboard still.

The whole point of the gaze keyboard control is to make keyboard layouts to be data files. This is a big requirement to support users of widely different abilities using different languages and having different needs. Even in Windows, the system keyboards use common logic and have different xml files internally for supporting different languages. When we prototyped this last, we had invented our own xml based format to support loading different keyboard layouts. When we did that, our format kept growing till it started resembling xaml. 

That is why I have taken the approach to shed a lot of our baggage and support specifying the keyboard layout directly in xaml. It is a big requirement for us to be able to treat keyboard layouts as data files and load them as needed.

The three layouts that are included don't even begin to scratch the surface of the variety required for keyboard layouts.